app/graphql/types/query_type.rb

```diff
@@ -208,9 +208,21 @@ def static_content_with_data_type(data_type, name, version = nil)
 
         static_content = find_static_content(static_contents, name, version)
 
-        return { content: static_content.content, name: name } if static_content.present?
-
-        { content: data_type == "html" ? "" : {}, name: "not found" }
+        return {
+          content: data_type == "html" ? "" : {},
+          name: "not found"
+        } if static_content.blank?
+
+        begin
+          content = static_content.content
+
+          return {
+            content: data_type == "html" ? content : JSON.parse(content),
```

Comment:
after merging #264 this should be rebased on master. the `static_content` should be returned at all and in case of json the `content` `.merge`ed.

Reply:
i will postpone it to #269